lib/uinta/formatter/datadog.ex

```diff
@@ -42,7 +42,7 @@ defmodule Uinta.Formatter.Datadog do
   rescue
     e ->
       IO.inspect(e)
```

Comment:
This should probably not be there anyway, but if we're inspecting the error on the next line, let's get rid of this

Would this error ever include sensitive data we don't want in the logs?